lib/crowbar/init/application.rb

```diff
@@ -315,6 +315,9 @@ class Application < Sinatra::Base
       # api_version "2.0"
       post "/api/upgrade/new" do
         api_constraint(2.0)
+        upgrade_status = ::Crowbar::UpgradeStatus.new(logger)
+        upgrade_status.start_step
```

Comment:
Currently, upgrade progress library will be off sync, pointing to a different step. We really need to care about disabling starting of steps out of order